js/about.js

```diff
@@ -0,0 +1,23 @@
+// navbar
+const menuBtn = document.querySelector('.menu-btn');
+const burger = document.querySelector('.burger');
+const header = document.querySelector('header');
+const menuNav = document.querySelector('.menu-navbar');
+let displayMenu = false;
+
+function toggleMenu() {
+  if (!displayMenu) {
+    burger.classList.add('open');
+    header.classList.add('open');
+    header.classList.add('open');
+    menuNav.classList.add('open');
+    displayMenu = true;
+  } else {
+    burger.classList.remove('open');
+    header.classList.remove('open');
+    menuNav.classList.remove('open');
+    displayMenu = false;
+  }
+}
+
+menuBtn.addEventListener('click', toggleMenu);
```

Comment:
- [optional] All these lines of code had already been implemented in `main.js`, I think you can remove it and just add `main.js` script to your `about.html` file. 👍  This will ensure that we follow DRY principle. Thanks. 👍